pkg/proxy/iptables/proxier.go

```diff
@@ -1308,48 +1288,18 @@ func (proxier *Proxier) syncProxyRules() {
 		if svcInfo.NodePort() != 0 && len(nodeAddresses) != 0 {
 			// Hold the local port open so no other process can open it
 			// (because the socket might open but it would never work).
-			lps := make([]netutils.LocalPort, 0)
 			for address := range nodeAddresses {
+				if utilproxy.IsZeroCIDR(address) {
+					address = ""
```

Comment:
Can you add a comment here that this is scoped to the specific IP family internally?  I had to go read code to convince myself.